py3_ruby/Dockerfile

```diff
@@ -18,12 +18,11 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -y update && \
 # Ruby installation
 ##################################################
 
-# update the rubygems system version to latest
-# RUN gem update --system 3.0.2
-
 # skip installing gem documentation
 RUN echo 'gem: --no-rdoc --no-ri --no-document' >> "/etc/gemrc" && \
-  gem install bundler -v 1.17
+  gem update --system && \
+  gem install bundler --no-document -v 1.17.3 && \
+  gem install bundler --no-document
```

Comment:
We should probably be explicit about this version as well. This is the version `2.x` of `bundler`, right?